templates/commands/render.go

```diff
@@ -361,15 +363,15 @@ func parseGoTmpl(tpl string) (*template.Template, error) {
 	return template.New("").Option("missingkey=error").Parse(tpl) //nolint:wrapcheck
 }
 
-func parseAndExecuteGoTmpl(m model.String, inputs map[string]string) (string, error) {
-	goTmpl, err := parseGoTmpl(m.Val)
+func parseAndExecuteGoTmpl(pos *model.ConfigPos, s string, inputs map[string]string) (string, error) {
```

Comment:
Nit: it's not clear what `s` is unless you read the code - it looks like it's the Go Template?

Reply:
I did some variable renaming to make this more clear.